### DISCUSSION TITLE
Flutter Web - PlatformException(channel-error, Unable to establish connection on channel., null, null)

### DISCUSSION THREAD

**Discussion by everfinal88** [firebase/flutterfire#10195] [Q&A]
Recently I integrated the firebase core and analytics into my flutter web project. Somehow when I run in localhost, it was working fine but after I deployed to production site then the error below shown in the console.
> PlatformException(channel-error, Unable to establish connection on channel., null, null)

Here is my firebase version in pubspec.yaml
>   firebase_core: ^2.4.1
>   firebase_analytics: ^10.1.0

I followed every instruction in the official document, I even reinstalled the entire firebase so I'm not sure what I missing here. Did anyone know how to deal with this issue? 
Any answer will be appreciated.

**Comment by Pineapples27** [comment]
It's probably too late to answer but the problem for me was the apiKey generated in firebase_options.dart was incorrect and had to be manually corrected
